app.js

```diff
@@ -3,14 +3,15 @@ const app=express();
 
 const {getTopics, 
     getArticles,
-    }
+    getArticleId}
     =require('./controllers/controllers');
 
 
 app.get('/api/topics',getTopics);
 
 app.get('/api/articles', getArticles);
 
+app.get('/api/articles/:article_id', getArticleId);
 
 app.use((err,req,res,next) => {
```

Comment:
This may be a good time to move your error handling out to a new file as you add in the extra handling for a 404 and 400 for the extra tests